### PR TITLE
Empty State Registration

### PR DESCRIPTION
**Pull request by premalrupnur**
<a href="https://gitpod.io/#https://github.com/Xcov19/mycovidconnect/pull/151"></a>

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/151)

### CODE REVIEW

**Comment by premalrupnur** [comment]
[
image
]
the page looks like above

**Comment by premalrupnur** [comment]
I think I have messed up with code formatting

File: src/containers/Search.jsx [hunk 1]
```diff
@@ -137,180 +139,212 @@ class Search extends Component {
...
+              <div className="text">
+                <h2>No results found for {city}</h2>
+                <br />
+                <span>
+                  Please register a request to enable service in your area{" "}
+                </span>
+              </div>
+
+              <div>
+                <a href="ESP_DOMAIN:8090/auth0/login/auth0>">
```
Comment (codecakes) [inline src/containers/Search.jsx:313]: ESP_PROXY_DOMAIN and ESP_PROXY_PORT

File: src/constants/index.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ const {
 	REACT_APP_COOKIEBOT_CONSENT_HEAD: COOKIEBOT_CONSENT_HEAD,
 	REACT_APP_COOKIEBOT_CONSENT_HEAD_DATA_CBID: COOKIEBOT_CONSENT_HEAD_DATA_CBID,
 	REACT_APP_TAWKTO: TAWKTO,
+	REACT_APP_ESP_DOMAIN:ESP_DOMAIN,
```
Comment (codecakes) [inline src/constants/index.js:21]: ive mentioned down 👇 where you've used to rename this;

**Review by codecakes — CHANGES_REQUESTED** [review]
Ive requested changes. 

In general, the git commits for any JIRA task should follow smart-commits. use that: https://confluence.atlassian.com/fisheye/using-smart-commits-960155400.html

Run `npm eslint:github-action` and `npm fmt`

File: src/App.css [hunk 3]
```diff
@@ -530,4 +561,4 @@ footer #cookieData .closebtn {
 	background-repeat: no-repeat;
 	background-size: 100px 100px;
 	background-position: center;
-}
+}
```
Comment (codecakes) [inline src/App.css:564]: EOF line space @premalrupnur

**Review by codecakes — APPROVED** [review]
Pls make changes and merge

**Review by ksr89 — APPROVED** [review]

File: src/constants/index.js [hunk 4]
```diff
@@ -18,6 +18,8 @@ const {
 	REACT_APP_COOKIEBOT_CONSENT_HEAD: COOKIEBOT_CONSENT_HEAD,
 	REACT_APP_COOKIEBOT_CONSENT_HEAD_DATA_CBID: COOKIEBOT_CONSENT_HEAD_DATA_CBID,
 	REACT_APP_TAWKTO: TAWKTO,
+	REACT_APP_ESP_PROXY_DOMAIN:ESP_PROXY_DOMAIN,
+	REACT_APP_ESP_PROXY_PORT:ESP_PROXY_PORT
```
Comment (premalrupnur) [inline src/constants/index.js:22]: @Akul same names have to be given in .env right I'll make those changes

**Pull request merged**